src/v/redpanda/admin/kafka_connections_service.h

```diff
@@ -47,6 +50,14 @@ class unordered_collector : public connection_collector {
         }
     }
 
+    ss::future<>
+    add_all(chunked_vector<proto::admin::kafka_connection> conns) final {
+        _connections.reserve(_connections.size() + conns.size());
+        co_await ssx::async_for_each(std::move(conns), [this](auto& conn) {
+            _connections.emplace_back(std::move(conn));
+        });
```

Comment:
This does not respect `_limit` like `add` and `bounded_priority_queue` do, is that intentional? I think you need to subrange `conns`.